Client/WarFare/UIVarious.cpp

```diff
@@ -699,6 +704,34 @@ void CUIKnights::RefreshButtonHandler(bool blBypassTime)
 	MsgSend_MemberInfoAll();
 }
 
+void CUIKnights::ClanPartyButtonHandler()
+{
+	int iSel = m_pList_CharIDs->GetCurSel();
+	if (iSel == -1)
+		return;
+
+	std::string szID;
+	m_pList_CharIDs->GetString(iSel, szID);
+
+	std::string myID = CGameProcedure::s_pPlayer->IDString();
+	std::string szMsg;
+
+	// Prevent inviting yourself
+	if (szID == myID)
```

Comment:
This needs to be a case-insensitive comparison, as it's dependent on the database data here, which doesn't tend to preserve the original character name's case, just how they were invited (AFAIK -- or the opposite, I don't remember). Officially they use `lstrcmpiA()` for these.